packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -2178,11 +2179,28 @@ export function flushPassiveEffects() {
   }
 }
 
-export function enqueuePendingPassiveEffectDestroyFn(
-  destroy: () => void,
+export function enqueuePendingPassiveHookEffectMount(
+  fiber: Fiber,
+  effect: HookEffect,
+): void {
+  if (deferPassiveEffectCleanupDuringUnmount) {
+    pendingPassiveHookEffectsMount.push(effect, fiber);
```

Comment:
Retaining both the effect and Fiber is kind of gross, particularly in the same array. I think we'd need to track them both *somewhere* though because of how we track and report errors.